src/main/java/ch/njol/skript/expressions/ExprColorOf.java

```diff
@@ -116,73 +175,115 @@ public Class<?>[] acceptChange(ChangeMode mode) {
 		if (mode != ChangeMode.SET && !getExpr().isSingle())
 			return null;
 
-		if (Entity.class.isAssignableFrom(returnType))
-			return CollectionUtils.array(Color.class);
-		else if (Block.class.isAssignableFrom(returnType))
-			return CollectionUtils.array(Color.class);
-		if (ItemType.class.isAssignableFrom(returnType))
-			return CollectionUtils.array(Color.class);
+		switch (mode) { // items/blocks/entities only accept these change modes
+			case RESET:
+			case DELETE:
+			case SET:
+				return CollectionUtils.array(Color.class);
+		}
+
 		return null;
```

Comment:
```suggestion
		switch (mode) { // items/blocks/entities only accept these change modes
			case RESET:
			case DELETE:
			case SET:
				return CollectionUtils.array(Color.class);
			default:
				return null;
		}
```
I think you can do this (can eventually be converted to a switch expression in the future)